apps/api/prisma/schema.prisma

```diff
@@ -33,7 +33,7 @@ model Intern {
   lastName          String
   email             String             @unique
   data              Json
-  hasInterviewRight Boolean            @default(true)
+  interviewStatus   InterviewStatus    @default(NoRight)
```

Comment:
jel nam bolje da je po defaultu ovdje `PickTerm` pa da mozemo samo prebacit u `NoRight` za one koji padnu dev ispit?

Reply:
U tom slučaju bi oni koji nisu napisali ispit mogli pristupiti intervju pickeru, šta ne? A ne želimo da mogu odabrat termin dok nisu položili npr dev ispit. Ali ovo bih da se određuje tako što samo devovima bude NoRight defaultno, ostalima PickTerm

Reply:
istina, može se samo postavit prilikom prijave na internship ovisno jel dev ili nije